docs/ContribOperators.md

```diff
@@ -4394,7 +4397,7 @@ This version of the operator has been available since version 1 of the 'com.micr
 </dl>
 
 
-### <a name="com.microsoft.QMoE"></a><a name="com.microsoft.qmoe">**com.microsoft.QMoE**</a>
+### <a name="com.microsoft.QMoE4Bits"></a><a name="com.microsoft.qmoe4bits">**com.microsoft.QMoE4Bits**</a>
```

Comment:
This is not backward compatible.
Shall we just add an attribute to QMoE to indicate whether it is 4 bit, 8 bit int quantization?

Reply:
let me change back to QMoE with attribute. the initial expectation was to add these two ops in before the release